TODO.md

```
@@ -0,0 +1,389 @@
+# TODO: 複数環境対応へのリファクタリング
+
+参考: https://zenn.dev/trifolium/articles/b3d88bbabcad2c
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)